docs/guided-tour.md

```diff
@@ -150,22 +162,29 @@ use a set of default parameters and iterate through sample names in Bash rather
 than create many individual JSON input files.
 
 ```bash
-sprocket run workflow.wdl defaults.json main.sample_name="Foo"
+sprocket run workflow.wdl defaults.json workflow.sample_name="Foo"
```

Comment:
Why is this now `workflow`? Isn't the target date still `main`?

Reply:
If you scroll up a few lines, the example JSON uses `workflow` instead of `main`. `sample_name` is also not a parameter for the `example.wdl` workflow.

But this is a bit confusing, I think it could be reworked a bit